server/server.js

```diff
@@ -3,6 +3,14 @@ var boot = require('loopback-boot');
 
 var app = module.exports = loopback();
 
+  //define environment variables for e-mail
+  if (!process.env.APP_URL || !process.env.APP_EMAIL || !process.env.APP_EMAILPW || !process.env.APP_SMTP) {
+    console.log("WARNING: Email environment variables not defined. App cannot send email.")
+    if (process.env.NODE_ENV == 'production'){
+      process.exit();
```

Comment:
We'll need to update our `app.json` config file for our Heroku review apps to continue working with this. I'll go ahead and make the changes.